packages/vue/src/composables/useBlockNumber.ts

```diff
@@ -75,7 +75,7 @@ export function useBlockNumber<
 
   const queryOptions = computed(() => {
     const {
-      chainId = configChainId.value,
+      chainId = configChainId,
```

Comment:
not reactive unless `.value` is removed, but messes with the type for `get*QueryOptions`

Reply:
need to figure out a nice way to use existing `get*QueryOptions` while also unwrapping these type of framework-specific values (in this case `ref`)

Related https://github.com/TanStack/query/pull/7390

Reply:
resolved https://github.com/wevm/wagmi/pull/3906/commits/ff73c95510ae256febf693f58a5bdb30214964f8